1224-user_administration/1424-administrator-configures-automatic-creator-role.feature

```diff
@@ -0,0 +1,14 @@
+@1424
+Scenario: Administrator configures automatic Creator role
+    Given that the User is logged in as Administrator
+    And they are on the User Administration Page
+    And they see a list of checkboxes in the Registrator section
+    And they see each checkbox has corresponding <Option> text
+    And they see that none of the checkboxes are selected
+    When they select an <Option> 
+    And they click the Save button
+    Then they see a Notification that the new values are persisted
```

Comment:
```suggestion
    Then they see a Notification that the <Option> has got the Creator role
```

Reply:
holder det ikke med en bekreftelse om at lagringen var vellykket?
Per nå skal det være to options (man kan velge begge), men i fremtiden kan det komme flere, så det kan bli en ganske lang melding til slutt

Reply:
@axthosarouris har et bra forslag til `Then`-setning 👇